### PR TITLE
Pi mobile dataset

### PR DESCRIPTION
**Pull request by pissac17**
data documentation for Mobile search tables regarding updated normalized app name

### CODE REVIEW

**Comment by pissac17** [comment]
documentation for Mobile search tables regarding updated normalized app name

**Comment by badboy** [comment]
you will have to add it to `SUMMARY.md` so that it shows up in the rendered book

File: src/datasets/search/mobile_search_clients_sources_daily/intro.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+We impute a `NULL` `engine` and `source` for pings with no search counts.
+This ensures users who never search are included in this dataset.
+
+This dataset is large.
+If you're querying this dataset from STMO,
+heavily limit the data you read using `submission_date` or `sample_id`.
+
+<!--
+#### Further Reading
+-->
```
Comment (badboy) [inline src/datasets/search/mobile_search_clients_sources_daily/intro.md:60]: Hm, I see this commented out part seems to be in several other pages as well. 
IMO we should just remove it here though.

```suggestion
```

**Review by skahmann3 — APPROVED** [review]
LGTM

**Review by badboy — CHANGES_REQUESTED** [review]
as said: you will have to add it to `SUMMARY.md` so that it shows up in the rendered book.

**Review by badboy — APPROVED** [review]

**Pull request merged**